templates/kubernetes/managed.html

```diff
@@ -17,7 +17,7 @@ <h1>Fully managed Kubernetes cloud, by Canonical</h1>
           <li>Monitoring and log management integration</li>
         </ul>
         <div>
-          <a href="{{ ASSET_SERVER_URL }}894857eb-DS_Enterprise-Kubernetes_screen-AW_11.17.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read the datasheet', 'eventLabel' : 'Read the datasheet - Top CTA' : undefined });" class="p-button--positive">Read the datasheet</a>
+          <a href="{{ ASSET_SERVER_URL }}223eab1a-canonical-kubernetes-enterprise-2018-25-07.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read the datasheet', 'eventLabel' : 'Read the datasheet - Top CTA' : undefined });" class="p-button--positive">Read the datasheet</a>
```

Comment:
Please drop the existing `onclick` event as they are now automatically generated on all links, with a new naming convention.